apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/QuestionCard.tsx

```diff
@@ -307,8 +324,7 @@ export default function QuestionCard({
                               maxLength={48}
                               placeholder={lastQuestion ? "Finish" : "Next"}
                               onChange={(e) => {
-                                if (e.target.value.trim() == "") e.target.value = "";
-                                updateQuestion(questionIdx, { buttonLabel: e.target.value });
+                                editNextButtonLabel(e.target.value);
```

Comment:
I'm not really satisfied that we loop through all questions `onChange` (with every character change). Can we also do this e.g. `onBlur`?

Reply:
That works 😊